contributors/design-proposals/job.md

```diff
@@ -79,8 +109,21 @@ type JobSpec struct {
     // job should be run with. Defaults to 1.
     Completions *int
 
+    // Optional duration in seconds relative to the startTime that the job may be active
```

Comment:
- [ ] I think the comments for exported fields should start with the name of the field: "ActiveDeadlineSeconds is an optional..."